### PR TITLE
feature: show warning message to alert users when there is a lambda execution in a proposal

### PR DESCRIPTION
**Pull request by rueshyna**
the change:
- refactor to add warning message

- only show warning message when signing and resolving with approval result. the same change is applied to settings for the warning of the change. 

when sign the proposal:
image

when reject the proposal:
image

when resolve the proposal (approve):
image

when resolve the proposal (reject):
image

cc: @tj825

### CODE REVIEW

**Review by JulesGuesnon — COMMENTED** [review]
Overall looks good, but I do have a small problem with the design. It really looks giant compared to the transactions above. I think it can be a good idea to try:
- adding more space between the warning and the transactions table
- reducing a bit the line height of the text

File: components/ProposalCard.tsx [hunk 1]
```diff
@@ -52,6 +57,18 @@ const ProposalCard = ({
...
+    for (let i = 0; i < rows.length; i++) {
+      metadataRender
+        ? tmp.push(
+            <RenderProposalContentMetadata key={i} content={content[i]} />
+          )
+        : tmp.push(<RenderProposalContentLambda key={i} data={rows[i]} />);
+    }
+    return tmp;
+  };
+
```
[the remark this answers is not in the document]
Comment (rueshyna) [reply components/ProposalCard.tsx]: no particular reasons.. modified as the suggestion and removed this.

**Comment by rueshyna** [comment]
> Overall looks good, but I do have a small problem with the design. It really looks giant compared to the transactions above. I think it can be a good idea to try:
> - adding more space between the warning and the transactions table
> - reducing a bit the line height of the text

I'm adding more warning messages this branch:
https://github.com/marigold-dev/tzsafe-ui/tree/cc%40warning_message_setting
I put messages in `li` tag. I'll also apply above suggestion with this branch. There will be easier not working on two branches. :)

**Review by JulesGuesnon — APPROVED** [review]
LGTM

**Pull request merged**